src/schema/ecommerce/types/order.js

```diff
@@ -60,7 +60,7 @@ export const OrderType = new GraphQLObjectType({
         _args,
         _context,
         { rootValue: { userByIDLoader } }
-      ) => userByIDLoader(userId),
+      ) => (userId ? userByIDLoader(userId) : null),
```

Comment:
I didn't understand why `userByIDLoader` from `partnerLoader` but with Order being null MP was making a request to gravity with `undifined` as ID and getting a gravity error.